dispatch/BESLog.cc

```diff
@@ -154,27 +180,29 @@ BESLog::~BESLog()
 {
     d_file_buffer->close();
     delete d_file_buffer;
-    d_file_buffer = 0;
+    d_file_buffer = nullptr;
```

Comment:
You can leave this in, but in a dtor, you don't need to set these to null since there is no way (unless there's a glaring compiler error) for them to be used again.